RecoMTD/TrackExtender/plugins/TrackExtenderWithMTD.cc

```diff
@@ -355,312 +362,321 @@ void TrackExtenderWithMTDT<TrackCollection>::produce( edm::Event& ev,
 
   auto tTrack = builder->build(track);
-  
+
   for (const DetLayer* ilay : layers) {
     const BoundDisk& disk = static_cast<const MTDRingForwardDoubleLayer*>(ilay)->specificSurface();
     const double diskZ = disk.position().z();
 
-    // get the outermost trajectory point on the track    
+    // get the outermost trajectory point on the track
     TrajectoryStateOnSurface tsos = tTrack.outermostMeasurementState();
-    if( tsos.globalPosition().z() * diskZ < 0 ) continue; // only propagate to the disk that's on the same side
-    find_hits_in_dets(hits,ilay,tsos,prop,*theEstimator,*hitbuilder,output);    
+    if (tsos.globalPosition().z() * diskZ < 0)
+      continue;  // only propagate to the disk that's on the same side
+    find_hits_in_dets(hits, ilay, tsos, prop, *theEstimator, *hitbuilder, output);
   }
   return output;
 }
 
-
-//below is unfortunately ripped from other places but 
+//below is unfortunately ripped from other places but
 //since track producer doesn't know about MTD we have to do this
-template<class TrackCollection>
+template <class TrackCollection>
 reco::Track TrackExtenderWithMTDT<TrackCollection>::buildTrack(const reco::Track& orig,
-							       const Trajectory& traj,
-							       const Trajectory& trajWithMtd,
-							       const reco::BeamSpot& bs,
-							       const MagneticField* field,
-							       const Propagator* thePropagator,
-							       bool hasMTD,
-							       float& pathLengthOut,
-							       float& tmtdOut,
-							       float& sigmatmtdOut) const {  
-
+                                                               const Trajectory& traj,
+                                                               const Trajectory& trajWithMtd,
+                                                               const reco::BeamSpot& bs,
+                                                               const MagneticField* field,
+                                                               const Propagator* thePropagator,
+                                                               bool hasMTD,
+                                                               float& pathLengthOut,
+                                                               float& tmtdOut,
+                                                               float& sigmatmtdOut) const {
   // get the state closest to the beamline
-  TrajectoryStateOnSurface stateForProjectionToBeamLineOnSurface = 
-    traj.closestMeasurement(GlobalPoint(bs.x0(),bs.y0(),bs.z0())).updatedState();
-  
-  if UNLIKELY(!stateForProjectionToBeamLineOnSurface.isValid()) {
-    edm::LogError("CannotPropagateToBeamLine")<<"the state on the closest measurement isnot valid. skipping track.";
-    return reco::Track();
-  }
+  TrajectoryStateOnSurface stateForProjectionToBeamLineOnSurface =
+      traj.closestMeasurement(GlobalPoint(bs.x0(), bs.y0(), bs.z0())).updatedState();
+
+  if
+    UNLIKELY(!stateForProjectionToBeamLineOnSurface.isValid()) {
```

Comment:
@smuzaffar @fabiocos 
was this break-up intentional or do we have something misconfigured in the clang-format?